.eslintrc.json

```diff
@@ -2,6 +2,9 @@
   "root": true,
   "extends": ["./node_modules/sanctuary-style/eslint-es3.json"],
   "env": {"node": true},
+  "globals": {
+    "Promise": "readonly"
```

Comment:
I'm using native Promises in the cjs version to align its output type with the mjs version. If you don't align these types, tooling tends to get confused, not to mention people.